### PR TITLE
Mahesh part 3

### PR DESCRIPTION
**Pull request by mahesh052002**
Practice examples

### CODE REVIEW

File: com/learning/mahesh/part3conditionalstatements/WhileLoopForPensionScheme.java [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+        int personB = 62;
+        int personC = 67;
+        int personD = 68;
+        int personE = 67;
+        int personF = 64;
+        int personG = 65;
+        int personH = 72;
+        int pensionagelimit = 65;
+
+        int[] people = {65, 62, 67, 68, 67, 64, 65, 72};
```
Comment (siddiswar) [inline com/learning/mahesh/part3conditionalstatements/WhileLoopForPensionScheme.java:16]: test review

File: com/learning/mahesh/part3conditionalstatements/WhileLoopForVoters.java [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+        int ageA = 20;
+        int ageB = 23;
+        int ageC = 21;
+        int ageD = 16;
+        int agelimit = 18;
+
+        int[] ages ={ageA, ageB, ageC, ageD };
+        int i = 0;
+
+        while (i < ages.length) {
```
Comment (siddiswar) [inline com/learning/mahesh/part3conditionalstatements/WhileLoopForVoters.java:14]: test review 2

**Review by siddiswar — CHANGES_REQUESTED** [review]

File: com/learning/mahesh/part3conditionalstatements/WhileLoopForVoters.java (hunk 2 above)
Comment (mahesh052002) [reply com/learning/mahesh/part3conditionalstatements/WhileLoopForVoters.java:14]: ok

File: com/learning/mahesh/part3conditionalstatements/WhileLoopForPensionScheme.java (hunk 1 above)
Comment (mahesh052002) [reply com/learning/mahesh/part3conditionalstatements/WhileLoopForPensionScheme.java:16]: ok

**Review by siddiswar — APPROVED** [review]

**Pull request merged**